.circleci/config.yml

```diff
@@ -196,8 +227,7 @@ jobs:
                 command: |
                   source scripts/utils.sh
                   tag=v$(extract_version_from_current_wd)
-                  # Newlines (<br/>) are necessary for better layout on Marketplace, but make the formatting worse on GitHub
-                  change_notes=$(sed '/^<br.>$/d' CHANGE-NOTES.html)
+                  change_notes=$(./gradlew getChangelog --no-header -q)
```

Comment:
Actually @MaciejG604 can it be simplified even more? `--no-header` skipped here, and just `$change_notes` with no `$tag` prefix passed as `$message`?

Reply:
Unfortunately no, this is what gets created: [Draft release](https://github.com/VirtusLab/git-machete-intellij-plugin/releases)

Reply:
☹️